consumer/src/test/kotlin/integrations/StreamsSinkProceduresIT.kt

```diff
@@ -0,0 +1,126 @@
+package integrations
+
+import org.apache.kafka.clients.producer.KafkaProducer
+import org.apache.kafka.clients.producer.ProducerConfig
+import org.apache.kafka.clients.producer.ProducerRecord
+import org.apache.kafka.common.serialization.ByteArraySerializer
+import org.apache.kafka.common.serialization.StringSerializer
+import org.junit.*
+import org.junit.rules.TestName
+import org.neo4j.kernel.impl.proc.Procedures
+import org.neo4j.kernel.internal.GraphDatabaseAPI
+import org.neo4j.test.TestGraphDatabaseFactory
+import org.testcontainers.containers.KafkaContainer
+import streams.procedures.StreamsSinkProcedures
+import streams.serialization.JSONUtils
+import java.util.*
+import kotlin.test.assertEquals
+import kotlin.test.assertFalse
+import kotlin.test.assertNotNull
+import kotlin.test.assertTrue
+
+
+class StreamsSinkProceduresIT {
+    companion object {
+        /**
+         * Kafka TestContainers uses Confluent OSS images.
+         * We need to keep in mind which is the right Confluent Platform version for the Kafka version this project uses
+         *
+         * Confluent Platform | Apache Kafka
+         *                    |
+         * 4.0.x	          | 1.0.x
+         * 4.1.x	          | 1.1.x
+         * 5.0.x	          | 2.0.x
+         *
+         * Please see also https://docs.confluent.io/current/installation/versions-interoperability.html#cp-and-apache-kafka-compatibility
+         */
+        private const val confluentPlatformVersion = "4.0.2"
```

Comment:
are we planning to test the other versions too?
or is that not needed?

Reply:
perhaps we can do it with a parameterized test or subclasses?

Reply:
We have the #51 that covers it